src/common/components/clusterDetail/KubeconfigDownload.tsx

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
-import { saveAs } from 'file-saver';
 import { Button, ButtonVariant } from '@patternfly/react-core';
-import { canDownloadKubeconfig } from '../hosts/utils';
-import { useAlerts } from '../AlertsContextProvider';
-import { Cluster } from '../../api/types';
-import { ocmClient } from '../../../ocm/api/axiosClient';
-import { getApiErrorMessage, handleApiError } from '../../../ocm/api/utils';
-import ClustersAPI from '../../../ocm/services/apis/ClustersAPI';
+import { saveAs } from 'file-saver';
 import { AxiosResponseHeaders } from 'axios';
-import { useTranslation } from '../../hooks/use-translation-wrapper';
 import { TFunction } from 'i18next';
+import { canDownloadKubeconfig } from '../hosts';
+import { useAlerts } from '../AlertsContextProvider';
+import { Cluster } from '../../api';
+import { getApiErrorMessage } from '../../utils';
+import { useTranslation } from '../../hooks/use-translation-wrapper';
+
+/* eslint-disable no-restricted-imports */
+import { ocmClient, handleApiError } from '../../../ocm/api';
+import ClustersAPI from '../../../ocm/services/apis/ClustersAPI';
+/* eslint-enable no-restricted-imports */
 
```

Comment:
The solution here is to pass the onHandle callback from OCM, same as CIM does.
Maybe add a TODO comment to make this change?

Reply:
Sorry, I missed this, and yes, it should be fixed as you mentioned.
I updated the PR, PTAL 

https://github.com/openshift-assisted/assisted-ui-lib/pull/1619/commits/37a19b785d744e255d33d85529e76fef623d00da
https://github.com/openshift-assisted/assisted-ui-lib/pull/1619/commits/a7810b970bf450dcbca0fbbcfb959b2c438a7e71